### PR TITLE
Add examples to Squeezing and Displacement Embeddings

### PR DESCRIPTION
**Pull request by ankit27kh**
**Description of the Change:**
add examples to Squeezing and Displacement Embeddings
**Benefits:**
better documentation
**Possible Drawbacks:**
none
**Related GitHub Issues:**
#1727 #1735

### CODE REVIEW

**Comment by ankit27kh** [comment]
Hi @antalszava, this should be complete.
These two were very similar; thus, I have used the same documentation for both while only changing the embedding itself.

File: pennylane/templates/embeddings/displacement.py [hunk 1]
```diff
@@ -41,6 +41,58 @@ class DisplacementEmbedding(Operation):
         c (float): value of the phase of all displacement gates if ``execution='amplitude'``, or
             the amplitude of all displacement gates if ``execution='phase'``
 
+    Example:
+
+        Depending on the ``method`` argument, the feature vector will be encoded in the phase or the amplitude.
+        The argument ``c`` will define the value of the other quantity.
+        The default values are :math:`0.1` for ``c`` and amplitude for ``method``.
```
Comment (antalszava) [inline pennylane/templates/embeddings/displacement.py:48]:
```suggestion
        The default values are :math:`0.1` for ``c`` and ``'amplitude'`` for ``method``.
```

File: pennylane/templates/embeddings/displacement.py [hunk 2]
```diff
@@ -41,6 +41,58 @@ class DisplacementEmbedding(Operation):
...
+
+            @qml.qnode(dev)
+            def circuit(feature_vector):
+                qml.DisplacementEmbedding(features=feature_vector, wires=range(3), method='phase', c=0.5)
+                qml.QuadraticPhase(0.1, wires=1)
+                return qml.expval(qml.NumberOperator(wires=1))
+
+            X = [1, 2, 3]
+
+        >>> print(circuit(X)
```
Comment (antalszava) [inline pennylane/templates/embeddings/displacement.py:86]:
```suggestion
        >>> print(circuit(X))
```

File: pennylane/templates/embeddings/squeezing.py [hunk 3]
```diff
@@ -42,6 +42,58 @@ class SqueezingEmbedding(Operation):
         c (float): value of the phase of all squeezing gates if ``execution='amplitude'``, or the
             amplitude of all squeezing gates if ``execution='phase'``
 
+    Example:
+
+        Depending on the ``method`` argument, the feature vector will be encoded in the phase or the amplitude.
+        The argument ``c`` will define the value of the other quantity.
+        The default values are :math:`0.1` for ``c`` and amplitude for ``method``.
```
Comment (antalszava) [inline pennylane/templates/embeddings/squeezing.py:49]:
```suggestion
        The default values are :math:`0.1` for ``c`` and ``'amplitude'`` for ``method``.
```

File: pennylane/templates/embeddings/squeezing.py [hunk 4]
```diff
@@ -42,6 +42,58 @@ class SqueezingEmbedding(Operation):
...
+
+            @qml.qnode(dev)
+            def circuit(feature_vector):
+                qml.SqueezingEmbedding(features=feature_vector, wires=range(3), method='phase', c=0.5)
+                qml.QuadraticPhase(0.1, wires=1)
+                return qml.expval(qml.NumberOperator(wires=1))
+
+            X = [1, 2, 3]
+
+        >>> print(circuit(X)
```
Comment (antalszava) [inline pennylane/templates/embeddings/squeezing.py:87]:
```suggestion
        >>> print(circuit(X))
```

**Review by antalszava — APPROVED** [review]
Looks good! :rocket: Applying some small suggestions, then ready to merge :ship:

File: pennylane/templates/embeddings/displacement.py [hunk 5]
```diff
@@ -41,6 +41,58 @@ class DisplacementEmbedding(Operation):
...
+
+        And, the resulting circuit is:
+
+        >>> print(qml.draw(circuit)(X))
+            0: ──D(0.5, 1)──────────┤
+            1: ──D(0.5, 2)──P(0.1)──┤ ⟨n⟩
+            2: ──D(0.5, 3)──────────┤
+
     Raises:
         ValueError: if inputs do not have the correct format
```
Comment (antalszava) [inline pennylane/templates/embeddings/displacement.py:97]: One detail: we'd like the `Example` section to come after `Raises`.
```suggestion
    Raises:
        ValueError: if inputs do not have the correct format

    Example:

        Depending on the ``method`` argument, the feature vector will be encoded in the phase or the amplitude.
        The argument ``c`` will define the value of the other quantity.
        The default values are :math:`0.1` for ``c`` and ``'amplitude'`` for ``method``.

        .. code-block:: python

            dev = qml.device('default.gaussian', wires=3)

            @qml.qnode(dev)
            def circuit(feature_vector):
                qml.DisplacementEmbedding(features=feature_vector, wires=range(3))
                qml.QuadraticPhase(0.1, wires=1)
                return qml.expval(qml.NumberOperator(wires=1))

            X = [1, 2, 3]

        >>> print(circuit(X))
            4.1215690638748494

        And, the resulting circuit is:

        >>> print(qml.draw(circuit)(X))
            0: ──D(1, 0.1)──────────┤
            1: ──D(2, 0.1)──P(0.1)──┤ ⟨n⟩
            2: ──D(3, 0.1)──────────┤

        Using different parameters:

        .. code-block:: python

            dev = qml.device('default.gaussian', wires=3)

            @qml.qnode(dev)
            def circuit(feature_vector):
                qml.DisplacementEmbedding(features=feature_vector, wires=range(3), method='phase', c=0.5)
                qml.QuadraticPhase(0.1, wires=1)
                return qml.expval(qml.NumberOperator(wires=1))

            X = [1, 2, 3]

        >>> print(circuit(X))
            0.23401288309122226

        And, the resulting circuit is:

        >>> print(qml.draw(circuit)(X))
            0: ──D(0.5, 1)──────────┤
            1: ──D(0.5, 2)──P(0.1)──┤ ⟨n⟩
            2: ──D(0.5, 3)──────────┤
```

Comment (antalszava) [inline pennylane/templates/embeddings/displacement.py:97]: One detail: we'd like the `Example` section to come after `Raises`.
```suggestion
    Raises:
        ValueError: if inputs do not have the correct format

    Example:

        Depending on the ``method`` argument, the feature vector will be encoded in the phase or the amplitude.
        The argument ``c`` will define the value of the other quantity.
        The default values are :math:`0.1` for ``c`` and ``'amplitude'`` for ``method``.

        .. code-block:: python

            dev = qml.device('default.gaussian', wires=3)

            @qml.qnode(dev)
            def circuit(feature_vector):
                qml.DisplacementEmbedding(features=feature_vector, wires=range(3))
                qml.QuadraticPhase(0.1, wires=1)
                return qml.expval(qml.NumberOperator(wires=1))

            X = [1, 2, 3]

        >>> print(circuit(X))
            4.1215690638748494

        And, the resulting circuit is:

        >>> print(qml.draw(circuit)(X))
            0: ──D(1, 0.1)──────────┤
            1: ──D(2, 0.1)──P(0.1)──┤ ⟨n⟩
            2: ──D(3, 0.1)──────────┤

        Using different parameters:

        .. code-block:: python

            dev = qml.device('default.gaussian', wires=3)

            @qml.qnode(dev)
            def circuit(feature_vector):
                qml.DisplacementEmbedding(features=feature_vector, wires=range(3), method='phase', c=0.5)
                qml.QuadraticPhase(0.1, wires=1)
                return qml.expval(qml.NumberOperator(wires=1))

            X = [1, 2, 3]

        >>> print(circuit(X))
            0.23401288309122226

        And, the resulting circuit is:

        >>> print(qml.draw(circuit)(X))
            0: ──D(0.5, 1)──────────┤
            1: ──D(0.5, 2)──P(0.1)──┤ ⟨n⟩
            2: ──D(0.5, 3)──────────┤
```

File: pennylane/templates/embeddings/squeezing.py [hunk 6]
```diff
@@ -42,6 +42,58 @@ class SqueezingEmbedding(Operation):
         c (float): value of the phase of all squeezing gates if ``execution='amplitude'``, or the
             amplitude of all squeezing gates if ``execution='phase'``
 
+    Example:
```
Comment (antalszava) [inline pennylane/templates/embeddings/squeezing.py:45]:
```suggestion
    Raises:
        ValueError: if inputs do not have the correct format

    Example:
```

File: pennylane/templates/embeddings/squeezing.py [hunk 7]
```diff
@@ -42,6 +42,58 @@ class SqueezingEmbedding(Operation):
...
+
+        And, the resulting circuit is:
+
+        >>> print(qml.draw(circuit)(X))
+            0: ──S(0.5, 1)──────────┤
+            1: ──S(0.5, 2)──P(0.1)──┤ ⟨n⟩
+            2: ──S(0.5, 3)──────────┤
+
     Raises:
         ValueError: if inputs do not have the correct format
```
Comment (antalszava) [inline pennylane/templates/embeddings/squeezing.py:98]:
```suggestion
```

File: pennylane/templates/embeddings/displacement.py (hunk 5 above)
Comment (antalszava) [inline pennylane/templates/embeddings/displacement.py:97]: One detail: we'd like the `Example` section to come after `Raises`.
```suggestion
```

File: pennylane/templates/embeddings/displacement.py [hunk 8]
```diff
@@ -41,6 +41,58 @@ class DisplacementEmbedding(Operation):
         c (float): value of the phase of all displacement gates if ``execution='amplitude'``, or
             the amplitude of all displacement gates if ``execution='phase'``
 
+    Example:
```
Comment (antalszava) [inline pennylane/templates/embeddings/displacement.py:44]: One detail: we'd like the `Example` section to come after `Raises`.
```suggestion
    Raises:
        ValueError: if inputs do not have the correct format

    Example:
```

File: pennylane/templates/embeddings/squeezing.py [hunk 9]
```diff
@@ -41,6 +41,60 @@ class SqueezingEmbedding(Operation):
...
+
+        And, the resulting circuit is:
+
+        >>> print(qml.draw(circuit)(X))
+            0: ──S(0.5, 1)──────────┤
+            1: ──S(0.5, 2)──P(0.1)──┤ ⟨n⟩
+            2: ──S(0.5, 3)──────────┤
 
     Raises:
         ValueError: if inputs do not have the correct format
```
Comment (antalszava) [inline pennylane/templates/embeddings/squeezing.py:100]:
```suggestion
```

File: pennylane/templates/embeddings/displacement.py [hunk 10]
```diff
@@ -41,8 +41,57 @@ class DisplacementEmbedding(Operation):
         c (float): value of the phase of all displacement gates if ``execution='amplitude'``, or
             the amplitude of all displacement gates if ``execution='phase'``
 
-    Raises:
-        ValueError: if inputs do not have the correct format
+    Example:
```
Comment (antalszava) [inline pennylane/templates/embeddings/displacement.py:44]:
```suggestion
    Raises:
        ValueError: if inputs do not have the correct format

    Example:
```
```

File: pennylane/templates/embeddings/squeezing.py [hunk 11]
```diff
@@ -41,9 +41,60 @@ class SqueezingEmbedding(Operation):
             ``'amplitude'`` uses the amplitude
         c (float): value of the phase of all squeezing gates if ``execution='amplitude'``, or the
             amplitude of all squeezing gates if ``execution='phase'``
-
     Raises:
```
Comment (antalszava) [inline pennylane/templates/embeddings/squeezing.py:44]:
```suggestion

    Raises:
```

**Pull request merged**